docs/assets/css/copy-button.css

```diff
@@ -0,0 +1,27 @@
+.copy-button {
+    position: absolute;
+    top: 0;
+    right: 0;
+    padding: 5px 10px;

```

Comment:
Use rem

Reply:
okay thankyou for your advice 
@MUzairS15